pnpm-lock.yaml

```diff
@@ -115,7 +115,7 @@ importers:
         specifier: ^8.57.0
         version: 8.57.0
       eslint-plugin-tailwindcss:
-        specifier: ^3.17.4
+        specifier: ^3.18.0
```

Comment:
This and the changes below are the result of updating the browsers list (caniuse), which needs to be done from time to time.